package.json

```diff
@@ -18,7 +18,6 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "prepare": "husky install",
-    "postinstall": "cd src/generator && npm install",
```

Comment:
When we're installing new package, we get a new error 
```
4610 error path /Users/n-shkaruba/Code/websql/node_modules/sql-autocomplete-parsers
4611 error command failed
4612 error command sh -c cd src/generator && npm install
4613 error sh: line 0: cd: src/generator: No such file or directory
4614 verbose exit 1
```

We don't include `src/generator` in our build [proof](https://www.npmjs.com/package/sql-autocomplete-parsers?activeTab=code), so postinstall doesn't work.